src/view/drag-handle/util/should-allow-dragging-from-target.js

```diff
@@ -16,15 +16,26 @@ export const interactiveTagNames: TagNameMap = {
   audio: true,
 };
 
-const isContentEditable = (parent: Element, current: ?Element): boolean => {
+const isAnInteractiveElement = (parent: Element, current: ?Element): boolean => {
   if (current == null) {
     return false;
   }
 
+  // Most interactive elements cannot have children. However, some can such as 'button'.
+  // See 'Permitted content' on https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button
+  // Rather than having two different functions we can consolidate our checks into this single
+  // function to keep things simple.
+  // There is no harm checking if the parent is has an interactive tag name even if it cannot have
+  // any children. We need to perform this loop anyway to check for the contenteditable attribute
+  const hasAnInteractiveTag: boolean = Boolean(interactiveTagNames[current.tagName.toLowerCase()]);
```

Comment:
Any reason why you're using the `Boolean` constructer to coerce to boolean over `!!`? `!!` is more performant but if this isn't being called a lot then it won't make a difference here. Micro optimisations aren't worth it :) https://jsperf.com/bool-vs-doublenot

Reply:
I think it is clearer

https://github.com/getify/You-Dont-Know-JS/blob/master/types%20%26%20grammar/ch4.md#explicitly-----boolean

Reply:
It also returns a value (it would return an `Boolean` object if we used `new Boolean(value)`

Reply:
Rather that using a double negative to cast - we are **super explicitly** casting to boolean :)